index.js

```diff
@@ -23,7 +23,8 @@ async function run() {
     process.env.CERTIFICATE_PASSWORD = core.getInput("certificate-password");
     process.env.OUTPUT_PATH = core.getInput("output-path");
     process.env.SCHEME = core.getInput("scheme");
-
+	process.env.DISABLE_TARGETS = core.getInput("disable-targets");
+	
```

Comment:
Please change from hard tab to soft tab.

```suggestion
    process.env.DISABLE_TARGETS = core.getInput("disable-targets");
```